### PR TITLE
release: create release branch for v0.22.2 to backport block parsing issue 

### PR DESCRIPTION
**Pull request by Roasbeef**

### CODE REVIEW

**Review by guggero — APPROVED** [review]
Looks good to me!
I checked that all important commits are in this branch vs. e3449998 which was the last commit used in `lnd v0.14.3-beta`. Everything that seems to be of importance is present. I assume 73f7eac9 isn't strictly required for our use cases?

**Comment by Roasbeef** [comment]
CI seems to be failing since a file that's assumed in the CI checks is no longer in the repo? 

edit: aaaand CI is gone all together?

**Comment by Roasbeef** [comment]
Tacked that commit on above in the end.

**Comment by Roasbeef** [comment]
Tests all pass for me locally, but it's using an ancient version of Go in the CI since that's what we had checked in at the time.

File: mempool/policy_test.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func TestCheckPkScriptStandard(t *testing.T) {
 	}
 }
 
-// TestDust tests the isDust API.
+// TestDust tests the IsDust API.
 func TestDust(t *testing.T) {
```
Comment (CHAINZ32) [inline mempool/policy_test.go:208]: Search

**Pull request merged**